maraboupy/MarabouNetworkNNet.py

```diff
@@ -141,18 +270,106 @@ def read_nnet(self, file_name):
             self.weights = weights
             self.biases = biases
 
-            # Convert input bounds from their original values to normalized values if normalization is not being incorporated
             if not self.normalize:
                 for i in range(self.inputSize):
                     self.inputMinimums[i] = (self.inputMinimums[i] - self.inputMeans[i]) / self.inputRanges[i]
                     self.inputMaximums[i] = (self.inputMaximums[i] - self.inputMeans[i]) / self.inputRanges[i]
 
+
+
```

Comment:
There are a few instances of double blank lines in this file that should be reduced to a single blank line to be consistent with the rest of Marabou files. I know it seems minor and picky, but I just want Marabou to have a clean, consistent, professional style. Generally it's best to follow the style of the existing code, or to follow the official python style guide: https://www.python.org/dev/peps/pep-0008/

Reply:
Fixed. 

I personally prefer double-blank lines around class members (I think it makes the code much more readable), but if the rest of maraboupy follows the PEP-08 convention, I am happy to go along -- I completely agree that a bit project it is important to have uniformity and consistency.